README.md

```diff
@@ -1,18 +1,64 @@
 # SSL
 
-Initial representation of the repo's structure:
+## What is what
 
-1. Firstly, you need to clone this repository near your datasets' folder. We are not going to store datasets in the repository, but we suppose that the needed data is downloaded to "datasets/dataset_name".
+* `sslearn` contains models and everything connected with machine learning.
+* `experementarium` contains functionality to launch some testing.
+* `config.py` parameters of the models that are to be tested.
 
-2. Each dataset has its own folder for testing inside "experiments/dataset_name" because we want to run tests independendently.
 
-3. You are able to upload your model into "models/your_model_name" for easier support, but it's not mandatory.
 
+## Benchmarks
 
-Workflow:
+### Adding new ones
 
-Once the new folder "experiments/dataset_name" appeared you need to update the repository, run the code for your model and add results to a table. 
+One needs to properly provide datareader to `experimentarium/data_react/rfuncs.py`, update `experimentarium/data_react/dataset2dir.json` and `experimentarium/data_react/dir2url.json`.
 
 
 
+## Testing
+
+Examine `experementarium/run.py` that supports command line arguments. Than run, supposing your current directory is `SSL`, command
+
+```bash
+python3 experementarium/run.py --arg1 value1 --arg2 value2 --arg3 value3
+```
+
+
+
+ By default it takes data from `../SSL/data` if there is no such folder it tries to download data from the internet.
+
+There is also a possibility to avoid typing arguments to variety command line arguments, just create file,, for instance,`../SSL/run.py` with the following content. 
+
+```python
+import sys
+
+repo_path = "SSL"
+sys.path.append(repo_path)
+
+from experimentarium.utils import ShellTestRunner  # noqa
+
+
+if __name__ == "__main__":
+    run_root = "SSL/experimentarium"
```

Comment:
@Vezzp 
Смог запустить только при run_root = "experimentarium"

Reply:
Странно, у меня работает при `SSL/experimentarium`, может, ты положил `run.py` в папку `SSL`, то есть внутрь репозитория, в то время как я положил его в одной директории с `SSL`?

Reply:
Да, я создал `run.py` в папке SSL. Просто из Readme мне показалось, что нужно так сделать:
```
There is also a possibility to avoid typing arguments to variety command line arguments, just create file,, for instance,../SSL/run.py with the following content.
```

Reply:
Окей. Будет здорово, если ты сформулируешь понятнее.